contracts/src/v0.8/upkeeps/CronUpkeep.sol

```diff
@@ -37,12 +36,11 @@ import {getRevertMsg} from "../utils/utils.sol";
 contract CronUpkeep is KeeperCompatibleInterface, KeeperBase, ConfirmedOwner, Pausable, Proxy {
   using EnumerableSet for EnumerableSet.UintSet;
 
-  event CronJobExecuted(uint256 indexed id, uint256 timestamp);
+  event CronJobExecuted(uint256 indexed id, bool success);
```

Comment:
timestamp is a useless field - you can get it from the block the tx was included in if needed